src/chains/ethereum/ethereum/src/provider.ts

```diff
@@ -359,13 +359,14 @@ export default class EthereumProvider
     >;
     if (promise instanceof PromiEvent) {
       promise.on("message", data => {
+        const serialized = JSON.parse(JSON.stringify(data));
```

Comment:
I don't think `serialized` is the right name here. Maybe `normalizedData` or `clonedData` would be better?

Reply:
I like `normalizedData`. Updated!